events.html

```diff
@@ -51,17 +52,36 @@
       </div>
     </nav>
 
-    <section id="calendar" class="content-section text-center">
-      <h2 class="section-heading text-uppercase">Upcoming Events</h2>
+    <section id="services" style="color: black;">
       <div class="container">
-        <iframe id="cal" src="https://calendar.google.com/calendar/embed?showTitle=0&amp;showPrint=0&amp;showCalendars=0&amp;height=600&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=maddie.gabes%40gmail.com&amp;color=%23711616&amp;ctz=America%2FToronto" style="border-width:0" width="1000" height="800" frameborder="0" scrolling="yes"></iframe>
+        <div class="row">
+          <div class="col-lg-12">
+            <h1 class="section-heading text-uppercase text-center">Upcoming Events</h1>
+            <h2 class="section-subheading text-uppercase">Computing Conferences and Meetups!</h2>
+            <ul>
+              <li><a href="https://ghc.anitab.org" target="_blank">Grace Hopper Celebration of Women in Computing! Sept 26-28, 2018</a> </li>
+              <li><a href="https://www.can-cwic.ca" target="_blank">The ACM Canadian Celebration of Women in Computing! Nov 2-3, 2018</a> </li>
+              <li><a href="http://cusec.net" target="_blank">Canadian University Software Engineering Conference! Date TBD, 2019</a> </li>
+              <li><a href="https://starcon.io" target="_blank">StarCon! Jan 19-20, 2019</a> </li>
+            </ul>
+            
+            <h2 class="section-subheading text-uppercase">Orientation Week!</h2>
+            <ul>
+              <li>Nerdy Trivia! Date TBD</li>
+              <li>Meet and Greet/Computing Fair, Date TBD</li>
```

Comment:
I believe the meet and greet/showcase is happening Tuesday September 4th

Reply:
and nerdy trivia is sunday September 2nd at 3pm

Reply:
Thanks!